src/Sulu/Bundle/DocumentManagerBundle/Bridge/DocumentInspectorFactory.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\DocumentManagerBundle\Bridge;
+
+use Sulu\Component\Content\Metadata\Factory\StructureMetadataFactoryInterface;
+use Sulu\Component\DocumentManager\DocumentInspectorFactoryInterface;
+use Sulu\Component\DocumentManager\DocumentManagerInterface;
+use Sulu\Component\DocumentManager\MetadataFactoryInterface;
+use Sulu\Component\DocumentManager\NamespaceRegistry;
+use Sulu\Component\DocumentManager\PathSegmentRegistry;
+use Sulu\Component\Webspace\Manager\WebspaceManagerInterface;
+
+class DocumentInspectorFactory implements DocumentInspectorFactoryInterface
```

Comment:
PHPDoc missing, and same concerns as in https://github.com/sulu/sulu-document-manager/pull/66#discussion_r59674644